requirements.yml

```diff
@@ -4,3 +4,23 @@
   name: openshift-contrib
 
 - src: ansible.kubernetes-modules
+
+- src: https://github.com/openstack/openstack-ansible-plugins
+  version: master 
+  name: openstack-ansible-plugins
+
+- src: https://github.com/openstack/ansible-role-k8s-tripleo
```

Comment:
This role needs ```meta/main.yaml``` or ci will fail

Reply:
This repo has been moved to tripleo-apb one: https://github.com/tripleo-apb/ansible-role-k8s-tripleo